### DISCUSSION TITLE
Field OPT-999 Dummy Tender Award Date

### DISCUSSION THREAD

**Discussion by kletersteegp** [OP-TED/eForms-SDK#904] [Q&A] [answered]
Hello,

What is field OPT-999 Dummy Tender Award Date. It is not in the CELEX excell sheet. It is prefilled with 2000-01-01. Should we overwrite this preset value? 

![image](https://github.com/OP-TED/eForms-SDK/assets/111063502/4cc45e56-86d1-45e8-b1a1-21bf06bff996)

Regards
Peter van de Kletersteeg

**Comment by YvesJo** [comment, marked as answer]
Hi,
This field has no other purpose than satisfy the XSD requirements; also no Business semantic is to be associated to it and everyone should use the same value "2000-01-01Z" as stated in the documentation (cf. https://docs.ted.europa.eu/eforms/latest/schema/competition-results.html#_notice_result )
